src/components/Main/Main.tsx

```diff
@@ -16,7 +16,7 @@ import run from '../../algorithms/run'
 import countryAgeDistributionData from '../../assets/data/country_age_distribution.json'
 import severityData from '../../assets/data/severityData.json'
 
-import countryCaseCounts from '../../assets/data/case_counts.json'
+import countryCaseCountData from '../../assets/data/case_counts.json'
```

Comment:
Remove the plural and add 'Data' suffix to match the convention when importing a json data module.